Reword exporters intro to not imply the collector is a viz+analysis backend (#3963)
open-telemetry/opentelemetry.io: layouts/shortcodes/docs/languages/exporters/intro.md

> To verify your telemetry exports correctly, one of the best things to do is send
> it to an [OpenTelemetry Collector](/docs/collector/). It is also generally a
> best practice to use the Collector in production. If you wisht to visualize your
> telemetry, you can also export it to a backend such as

Reviewer: ```suggestion
To ensure that your telemetry is exporting correctly, send
it to the [OpenTelemetry Collector](/docs/collector/). It is also considered a
best practice to use the Collector in production. To visualize your
telemetry, export it to a backend such as
```

> To ensure that your telemetry is exporting correctly, send it to the
> [OpenTelemetry Collector](/docs/collector/). It is also considered a best
> practice to use the Collector in production. To visualize your telemetry, export
> it to a backend such as [Jaeger](https://jaegertracing.io/),
> [Zipkin](https://zipkin.io/), [Prometheus](https://prometheus.io/), or a

Reviewer 2: ```suggestion
Send telemetry to the [OpenTelemetry Collector](/docs/collector/) to
make sure it's exported correctly. Using the Collector in production
environments is a best practice. To visualize your telemetry, export
it to a backend such as [Jaeger](https://jaegertracing.io/),
[Zipkin](https://zipkin.io/), [Prometheus](https://prometheus.io/), or a
```

Reviewer: Thank you @theletterf, that's much better! 🙏 
(I wasn't pleased with the text, but didn't have the inspiration to improve it.)
